worlds/lamulana/LmFlags.py

```diff
@@ -1,197 +1,195 @@
-  "slushfund_give_anchor": 247,
-  "xelpud_xmailer": 364,
-  "xelpud_talisman": 369,
-  "xelpud_pillar": 370,
-  "xelpud_mulana_talisman": 371,
-  "xelpud_conversation_tree": 480,
-  "nebur_guardian": 490,
-  "xelpud_howling_wind": 1049
+    "slushfund_give_pepper": 245,
+    "slushfund_give_anchor": 247,
+    "xelpud_xmailer": 364,
+    "xelpud_talisman": 369,
+    "xelpud_pillar": 370,
+    "xelpud_mulana_talisman": 371,
+    "xelpud_conversation_tree": 480,
+    "nebur_guardian": 490,
+    "xelpud_howling_wind": 1049
 }
 
 RCD_OBJECTS = {
-  "trigger_dais": 0x8,
-  "moving_texture": 0xa,
-  "flag_timer": 0xb,
-  "room_spawner": 0xe,
-  "crusher": 0x11,
-  "hitbox_generator": 0x12,
-  "lemeza_detector": 0x14,
-  "counterweight_platform": 0x33,
-  "chest": 0x2c,
-  "naked_item": 0x2f,
-  "vimana": 0x71,
-  "texture_draw_animation": 0x93,
-  "use_item": 0x9c,
-  "scannable": 0x9e,
-  "grail_point": 0x9f,
-  "language_conversation": 0xa0,
-  "fairy_keyspot": 0xa7,
-  "explosion": 0xb4,
-  "instant_item": 0xb5,
-  "scan": 0xc3
+    "trigger_dais": 0x8,
+    "moving_texture": 0xa,
+    "flag_timer": 0xb,
+    "room_spawner": 0xe,
+    "crusher": 0x11,
+    "hitbox_generator": 0x12,
+    "lemeza_detector": 0x14,
+    "counterweight_platform": 0x33,
+    "chest": 0x2c,
+    "naked_item": 0x2f,
+    "vimana": 0x71,
+    "texture_draw_animation": 0x93,
+    "use_item": 0x9c,
+    "scannable": 0x9e,
+    "grail_point": 0x9f,
+    "language_conversation": 0xa0,
+    "fairy_keyspot": 0xa7,
+    "explosion": 0xb4,
+    "instant_item": 0xb5,
+    "scan": 0xc3
 }
 
 TEST_OPERATIONS = {
-  "eq": 0x0,
-  "lteq": 0x1,
-  "gteq": 0x2,
-  "andnz": 0x3,
-  "ornz": 0x4,
-  "xornz": 0x5,
-  "zero": 0x6,
-  "neq": 0x40,
-  "gt": 0x41,
-  "lt": 0x42,
-  "andz": 0x43,
-  "orz": 0x44,
-  "xorz": 0x45,
-  "nz": 0x46
+    "eq": 0x0,
+    "lteq": 0x1,
+    "gteq": 0x2,
+    "andnz": 0x3,
+    "ornz": 0x4,
+    "xornz": 0x5,
+    "zero": 0x6,
+    "neq": 0x40,
+    "gt": 0x41,
+    "lt": 0x42,
+    "andz": 0x43,
+    "orz": 0x44,
+    "xorz": 0x45,
+    "nz": 0x46
 }
 
 WRITE_OPERATIONS = {
-  "assign": 0x0,
-  "add": 0x1,
-  "sub": 0x2,
-  "mult": 0x3,
-  "div": 0x4,
-  "and": 0x5,
-  "or": 0x6,
-  "xor": 0x7
+    "assign": 0x0,
+    "add": 0x1,
+    "sub": 0x2,
+    "mult": 0x3,
+    "div": 0x4,
+    "and": 0x5,
+    "or": 0x6,
+    "xor": 0x7
 }
 
+
 def grail_flag_by_zone(zone, frontside):
-  match zone:
-    case 0:
-      return GLOBAL_FLAGS["grail_tablet_guidance"]
-    case 1:
-      return GLOBAL_FLAGS["grail_tablet_surface"]
-    case 2:
-      return GLOBAL_FLAGS["grail_tablet_mausoleum"]
-    case 3:
-      return GLOBAL_FLAGS["grail_tablet_sun"]
-    case 4:
-      return GLOBAL_FLAGS["grail_tablet_spring"]
-    case 5:
-      return GLOBAL_FLAGS["grail_tablet_inferno"]
-    case 6:
-      return GLOBAL_FLAGS["grail_tablet_extinction"]
-    case 7:
-      if frontside:
-        return GLOBAL_FLAGS["grail_tablet_twin_front"]
-      else:
+    if zone == 0:
```

Comment:
I don't agree with this change. Even though match supports more complex pattern matching, it also supports literal matching. In this case, it significantly reduces the repetition of the variable name `zone` which I think improves legibility and maintainability if the variable were to change in the future